Automattic/wp-calypso
# Remove pressable banner from pricing table

This PR is an action item from p58i-gcf-p2 to remove the presssable banner from the pricing page

## Proposed Changes

* Remove code related to the pressable banner

## Testing Instructions



* Go to `/start/plans`
* Verify the Pressable banner is no longer present


![image](https://github.com/Automattic/wp-calypso/assets/47489215/0bf00f34-93a9-45b7-b855-046fc73eb811)


## Pre-merge Checklist
